src/realtime/realtime-hapi-plugin.ts

```diff
@@ -128,41 +129,12 @@ export class RealtimeHapiPlugin {
       },
     });
 
-    // TODO: Used for testing, should be removed in production
-    server.route({
-      method: 'POST',
-      path: '/events/{teamId}',
-      handler: {
-        async: this.postHandler,
-      },
-      config: {
-        bind: this,
-        validate: {
-          params: {
-            teamId: Joi.number().required(),
-          },
-        },
-      },
-    });
-
     next();
 
   }
 
   public readonly register: HapiRegister;
 
-  private async postHandler(request: Hapi.Request, reply: Hapi.IReply) {
-    try {
-      const isPersisted = await this.eventBus.post(request.payload);
-      reply(JSON.stringify(request.payload, null, 2))
-        .code(isPersisted ? 500 : 200);
-
-    } catch (err) {
-      this.logger.error('Error:', err);
-      reply(err);
-    }
-  }
-
   private pingEvent() {
     return {
```

Comment:
This function should be changed to this instead of hard-casting:

```
    const event: PersistedEvent<any> = {
      type: 'CONTROL_PING',
      id: '0',
      streamRevision: 0,
      teamId: 0,
      created: moment(),
      payload: 0,
    };

    return event;
```